kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-core/src/test/java/org/kie/workbench/common/services/backend/compiler/ClassLoaderProviderTest.java

```diff
@@ -161,6 +159,7 @@ public void getClassloaderFromAllDependenciesTestComplex() {
     }
 
     @Test
+    @Ignore //https://issues.redhat.com/browse/AF-2892
```

Comment:
Can we use the Parameter on all `@Ignore` annotated tests
```suggestion
    @Ignore("https://issues.redhat.com/browse/AF-2892")
```